warema_wms/shade.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, wms_ctrl: WmsController, room, channel,
         self.time_between_cmds = time_between_cmds
         self.num_retries = num_retries
         self.position = position  # 0 for open 100 for closed
+        self.tilt_position = tilt_position    # lets see what happens now # 30=tilt open 100=tilt closed, 6 Raffstores show different values here, from 20 - 28 & 101 - 104
```

Comment:
```suggestion
        self.tilt_position = tilt_position    # 30=tilt open 100=tilt closed, 6 Raffstores show different values here, from 20 - 28 & 101 - 104
```

I would limit the comments to technical content

Reply:
True, overseen that